app/com/arpnetworking/database/h2/triggers/BaseUpdateEtagTrigger.java

```diff
@@ -56,9 +60,21 @@ public void init(
     @Override
     @SuppressFBWarnings("SQL_PREPARED_STATEMENT_GENERATED_FROM_NONCONSTANT_STRING")
     public void fire(final Connection conn, final Object[] oldRow, final Object[] newRow) throws SQLException {
-        try (PreparedStatement stmt = conn.prepareStatement(_queryStatement)) {
+        final int updateCount;
+        final int orgId = (int) (oldRow != null ? oldRow[_orgIndex] : newRow[_orgIndex]);
+        Logger.info("Updating etag for organization " + orgId);
+        try (final PreparedStatement stmt = conn.prepareStatement(_updateStatement)) {
+            stmt.setInt(1, orgId);
```

Comment:
We should use named not positional arguments. I believe both are supported by EBean.